errors/index.js

```diff
@@ -1,6 +1,6 @@
 exports.handleCustomErrors = (err, req, res, next) => {
-  if (err.status && err.msg) {
-    res.status(err.status).send({ msg: err.msg });
+  if (err.status === 404 && err.msg === "Resource not found") {
+    res.status(404).send({ msg: "PATH REQUESTED NOT FOUND" });
```

Comment:
I think the previous setup may have been preferable here - remember that for this error handler, you have created the custom error object anyway so should be able to reuse elsewhere